dataprofiler/profilers/profile_builder.py

```diff
@@ -1887,7 +1907,10 @@ def report(self, report_options: dict = None) -> dict:
     def _get_unique_row_ratio(self) -> float:
         """Return unique row ratio."""
         if self.total_samples:
```

Comment:
what if unique_row_ratio is disabled? this would return a result which may not be true to the dataset. Tests would be required here.